src/components/ActionMenu/MenuItems/AddNew.js

```diff
@@ -0,0 +1,41 @@
+import React, {
+  memo,
+} from 'react';
+import { FormattedMessage } from 'react-intl';
+import PropTypes from 'prop-types';
+
+import { Icon, Button } from '@folio/stripes/components';
+
+import sharedCss from '../../../shared.css';
+import { createLayerURL } from '../../../utils';
+import { LAYER_TYPES } from '../../../utils/constants';
+
+export const AddNew = memo(props => {
```

Comment:
Please, add a target purpose of a component to a component name (For example, AddNewItem or something like this). For now, it sounds like a function name. It looks like it can be applied for Default and ExportSelected components.

Reply:
It's not needed to do so for the following reason:
1 - It is obvious that it is a component but not a function
2 - We use constants to get control names here
3 - It probably will be changed within the refactoring spike efforts